translator/tocwconfig/sampleConfig/compass_linux_config.json

```diff
@@ -15,6 +15,7 @@
     "deployment.environment": "agent-level-environment"
   },
   "logs": {
+    "concurrency": 1,
```

Comment:
Set to 1 to avoid having to update all the sample configurations to check the default concurrency value. The default concurrency value has a unit test in the translator changes.